bundles/org.openhab.binding.ism8/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="ism8" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Ism8 Binding</name>
+	<description>This is the binding for Wolf Ism8.</description>
```

Comment:
You could add "heating system" or so to give users a clue what this is about, who never got in touch with this.

Reply:
Yes, updated. This is the binding for the Ism8 card used for Wolf heating systems or other Wolf eBus devices.